packages/hurl/src/main.rs

```diff
@@ -275,6 +275,7 @@ pub mod tests {
                 errors: vec![],
                 time_in_ms: 0,
                 compressed: false,
+                max_filesize: 0,
```

Comment:
I don't see why we should add `max_filesize` here. Entry result structure should not changed.